README.md

```diff
@@ -1,39 +1,33 @@
 # MIxS: Minimum Information about any (X) Sequence
 
-⚠️ ⚠️ ⚠️ 
+MIxS is a specification for the Minimum Information about any (X) Sequence standard.
 
-The GSC is preparing for the MIxS 6.2.0 release. Thanks to everyone who has contributed in any way!
+Starting from v6 the MIxS standard is represented in and maintained using the [LinkML](https://linkml.io/linkml/) framework. LinkML uses YAML to define schemas and data models. You can refer to the [user-facing](#repository-structure)/[developer-focused](#developer-documentation) sections describing the repository structure to understand where to find these YAML files that are describing the MIxS standard.
 
-The new release primarily consists of converting a spreadsheet expressing MIxS 6.1 into a [LinkML artifact, which now serves as the source of truth](https://github.com/GenomicsStandardsConsortium/mixs/blob/main/src/mixs/schema/mixs.yaml). 
-We have also started to follow the [LinkML cookiecutter layout](https://github.com/linkml/linkml-project-cookiecutter).
-MIxS users should see very few changes in the content, although legacy term attributes like "Definition", "Expected value", "Value syntax" and "Section" have been broken down into their LinkML equivalents, 
-like `description`, `range`, `pattern` and `in_subset`.
+The public-facing web documentation site can be browsed at: https://genomicsstandardsconsortium.github.io/mixs/
 
-We have [some known issues](https://github.com/GenomicsStandardsConsortium/mixs/issues?q=is%3Aissue+is%3Aopen+label%3A6.1.1-%3E6.2.0), and you may find others. [Please report them](https://github.com/GenomicsStandardsConsortium/mixs/issues) including text snippets and or screen shots!
+MIxS is divided into Checklists and Extensions (previously called Packages).
```

Comment:
what does "divided into mean"

I would suggest: Most MIxS users will primary relate to MIxS term, which describe the samples and their sequences. But terms aren't free-standing. They belong to Checklists, Extensions (...) and combinations of those two. That means we can say "for this kind of sample, you mus tell us bout..."